perun-web-gui/src/main/java/cz/metacentrum/perun/webgui/tabs/groupstabs/GroupMembersTabItem.java

```diff
@@ -132,7 +132,9 @@ public Widget draw() {
 
 		final CustomButton searchButton = TabMenu.getPredefinedButton(ButtonType.SEARCH, ButtonTranslation.INSTANCE.searchMemberInGroup());
 		final CustomButton listAllButton = TabMenu.getPredefinedButton(ButtonType.LIST_ALL_MEMBERS, ButtonTranslation.INSTANCE.listAllMembersInGroup());
-		if (!session.isVoAdmin(group.getVoId()) && !session.isGroupAdmin(groupId)) findMembers.setCheckable(false);
+		if (!session.isVoAdmin(group.getVoId()) && !session.isGroupAdmin(groupId) && !group.isCoreGroup()) {
```

Comment:
I believe here it should be ``if ((cond1 && cond2 ) || group.isCoreGroup()) { disable }`` since if not admin or is a core group then don't allow to select members.